package NPrsLog.FactorStorageCompressionProto;

message TCompressedWebData {
    repeated string DocIds = 1;
    repeated string Urls = 2;
    optional bytes WebFeaturesIds = 3;
    optional bytes WebMetaFeaturesIds = 4;
    repeated string Features = 5;
}
